ocean_lib/data_provider/data_service_provider.py

```diff
@@ -11,6 +11,7 @@
 from collections import namedtuple
 from json import JSONDecodeError
 
+import requests
```

Comment:
This should be part of the imports in the 3rd party libraries above. Eg remove the space above this line, and add a space below, and you're good.